### PR TITLE
Remove pending tasks scope

### PR DESCRIPTION
**Pull request by jonasfj**
For quick review...

### CODE REVIEW

File: routes/v1.js [hunk 1]
```diff
@@ -1631,28 +1631,21 @@ api.declare({
...
   description: [
-    "Documented later...",
-    "This probably the end-point that will remain after rewriting to azure",
-    "queue storage...",
+    "Get an approximate number of pending tasks for the given `provisionerId`",
+    "and `workerType`.",
     "",
+    "The underlying Azure Storage Queues only promises to give us an estimate.",
+    "Furthermore, we cache the result in memory for 20 seconds. So consumers",
+    "should be no means expect this to be an accurate number.",
```
Comment (gregarndt) [inline routes/v1.js]: "by no means"

File: routes/v1.js [hunk 2]
```diff
@@ -1631,28 +1631,21 @@ api.declare({
...
   output:     'pending-tasks-response.json#',
   title:      "Get Number of Pending Tasks",
   description: [
-    "Documented later...",
-    "This probably the end-point that will remain after rewriting to azure",
-    "queue storage...",
+    "Get an approximate number of pending tasks for the given `provisionerId`",
+    "and `workerType`.",
     "",
+    "The underlying Azure Storage Queues only promises to give us an estimate.",
```
Comment (gregarndt) [inline routes/v1.js]: Hrm, do we need to provide this level of details in our user facing documentation?  Seems like users don't care what Azure storage queue is or why this is just an estimate.

**Comment by gregarndt** [comment]
lgtm

File: routes/v1.js (hunk 2 above)
Comment (jonasfj) [inline routes/v1.js]: good point will drop that reference...

**Comment by jonasfj** [comment]
oh, I forgot to merge this...

**Pull request merged**